pages/common/pueue-completions.md

```diff
@@ -1,9 +1,9 @@
 # pueue completions
 
-> Generates shell completion files for bash, elvish, fish, powershell, and zsh.
+> Generates shell completion files for Bash, elvish, fish, PowerShell, and zsh.
```

Comment:
```suggestion
> Generates shell completion files for Bash, Elvish, Fish, PowerShell, and ZSH.
```

Reply:
```suggestion
> Generates shell completion files for Bash, elvish, fish, PowerShell, and Zsh.
```

Don't mind this, just adding the comment, so I can merge the version with Zsh instead of ZSH.